web/src/js/ducks/ui/flow.js

```diff
@@ -87,9 +88,31 @@ export function updateEdit(update) {
     return { type: UPDATE_EDIT, update }
 }
 
-export function stopEdit(flow) {
+export function uploadContent(flow, content, type){
     return (dispatch) => {
-        dispatch(flowsActions.update(flow, flow)).then(() => {
+            dispatch(flowsActions.updateContent(flow, content, type)).then( () => {
```

Comment:
STOP_EDIT: because i thought that after a content upload it would be good to close the edit mode if it is open. but it would be better when the editor is refreshed with the new content